.whatsnew.release.json

```diff
@@ -0,0 +1,80 @@
+{
+    "$schema": "https://whatsnewapi.azurewebsites.net/schema",
+    "docSetProductName": ".NET",
+    "rootDirectory": "docs/",
+    "docLinkSettings": {
+        "linkFormat": "relative",
+        "relativeLinkPrefix": "../"
+    },
+    "inclusionCriteria": {
+        "labels": [
+            "label:\":checkered_flag: Release: .NET 7\""
+        ],
+        "minAdditionsToFile": 1
+    },
+    "navigationOptions": {
+        "maximumNumberOfArticles": 3,
+        "tocParentNode": "What's new",
+        "repoTocFolder": "docs/whats-new",
+        "indexParentNode": ".NET Release updates",
+        "repoIndexFolder": "docs/whats-new"
+    },
```

Comment:
@gewarren This section defines where the index.yml and toc.yml edits are performed. In addition, this defines how many articles in that section are included.